### DISCUSSION TITLE
What is this "gallery"?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#538] [Q&A] [answered]
I have been seeing alot of commits towards gallery. What is it? Is it like Tailwind Components/Material UI premade components?

**Comment by rtivital** [comment, marked as answer]
Yes, it's an upcoming project with prebuilt components

**Reply by dukesx** [reply]
Will they be Open source or proprietary licensed ?

**Reply by rtivital** [reply]
Open source

**Reply by murbanowicz** [reply]
That’s awesome! 
Is it deployed in any way or not yet?

**Reply by rtivital** [reply]
No, it's not deployed, it's planned to go live in Q1 2022

**Reply by sahrohit** [reply]
Do we have a fixed date for the release of Mantine Gallery yet?

**Reply by rtivital** [reply]
Gallery will be out with 4.0 release. 4.0 release will be out when it's ready (no fixed date).
